src/js/actions/menu.js

```diff
@@ -570,6 +570,181 @@ define(function (require, exports) {
         writes: [locks.JS_PREF],
         transfers: [preferencesActions.setPreference]
     };
+    
+    /**
+     * Debug-only method to toggle Descriptor Event logging
+     * 
+     * @param {Object} options
```

Comment:
In this case, the correct type expression is `object`. You should only use upper case `Object` when you're describing an object which is being used as a map, and hence write something like `Object.<S,T>`.